include/tvm/tir/expr.h

```diff
@@ -829,6 +829,7 @@ class CallNode : public PrimExprNode {
   static constexpr const char* glsl_texture_store = "glsl_texture_store";
   static constexpr const char* prefetch = "prefetch";
   static constexpr const char* isnan = "isnan";
```

Comment:
Is it worth adding an `isinf` operator to this PR? That way we can check for both halves of isfinite separately if needed. I'm not sure if you'd ever need to check for just infinity rather than both nan and infinity though.

Reply:
Added support for isinf as well.